docs/modules/project/pages/governance.adoc

```diff
@@ -0,0 +1,38 @@
+= Asciidoctor.js Governance
+:keywords: governance
+:uri-npmjs-asciidoctorjs: https://www.npmjs.com/package/asciidoctor.js
+:uri-gh-asciidoctorjs: https://github.com/asciidoctor/asciidoctor.js
+
+== Core team
+
+The Asciidoctor.js core team consists of 8 members.
+Each member has been part of the https://github.com/asciidoctor[Asciidoctor community] for several years,
+and had to earn the trust of the other members before joining the core team.
+
+If a new member wanted to join the core team, we would discuss it with the community to evaluate if the new member is trustworthy.
+
+=== GitHub
+
+Maintainers (write access)::
+* Dan Allen -- https://github.com/mojavelinux[@mojavelinux]
+* Guillaume Grossetie -- https://github.com/mogztter[@mogztter]
+
+Members::
+* Thomas Parisot -- https://github.com/oncletom[@oncletom]
+* Sylvain Leroux -- https://github.com/s-leroux[@s-leroux]
+* Anthonny Quérouil -- https://github.com/anthonny[@anthonny]
+
+Only the maintainers have the right to push on the {uri-gh-asciidoctorjs}[GitHub repository].
+
+
+=== npmjs
+
+Maintainers (write access)::
+* Dan Allen -- https://github.com/mojavelinux[@mojavelinux]
+* Olivier Bilodeau -- https://github.com/obilodeau[@obilodeau]
```

Comment:
Do I really need npmjs write access? I think you guys are going to do fine without me. Unless it's somehow a group and related to asciidoctor-reveal.js I would drop myself out of that. Based on the security principle of "least privilege"

Reply:
That's a good point, I think we should reduce the number of people with write access to just 2 (or maybe 3).
Especially given that releases are now performed by Travis using my token. So if a maintainer with write access to GitHub want to do a release, he can create a tag and push it to trigger a release. So 2 or 3 maintainers on npmjs is enough to recover access or perform a manual release.